.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,75 @@
+{
+  "name": "config",
+  "build": {
+    "cacheFrom": "ghcr.io/marinatedconcrete/config-devcontainer:latest",
+    "context": "..",
+    "dockerfile": "Dockerfile"
+  },
+  "customizations": {
+    "vscode": {
+      "extensions": [
+        "esbenp.prettier-vscode",
+        // Enable VSCode to find the prettier package to use when formatting
+        "arcanis.vscode-zipfs",
+        "github.vscode-github-actions",
+        "ms-kubernetes-tools.vscode-kubernetes-tools",
+        "nefrob.vscode-just-syntax",
+        "redhat.vscode-yaml"
+      ],
+      "settings": {
+        "prettier.useEditorConfig": false,
+        "search.exclude": {
+          "**/.yarn": true,
+          "**/.pnp.*": true
+        },
+        "prettier.prettierPath": ".yarn/sdks/prettier/index.js"
+      }
+    }
+  },
+  "features": {
+    "ghcr.io/devcontainers/features/docker-in-docker:2": {
+      // TODO: figure out docker versioning story here
+    },
+    "ghcr.io/devcontainers/features/kubectl-helm-minikube:1": {
+      // renovate: datasource=github-releases depName=kubernetes/kubernetes
+      "version": "1.32.0"
+    },
+    "ghcr.io/devcontainers/features/git:1": {
+      "version": "os-provided"
+    },
+    "ghcr.io/devcontainers/features/github-cli:1": {
+      // renovate: datasource=github-releases depName=cli/cli
+      "version": "2.64.0"
+    },
+    "ghcr.io/dhoeric/features/hadolint:1": {
+      // renovate: datasource=github-releases depName=hadolint/hadolint
+      "version": "2.12.0"
+    },
+    "ghcr.io/devcontainers/features/node:1": {
+      // TODO: figure out how to version node
```

Comment:
From the docs, it looks like we can just supply `version` here: https://github.com/devcontainers/features/tree/main/src/node

I don't know if you care to also version npm, etc

Reply:
For this and the other one, I wasn't sure what the renovate line should look like. Separately, the risk of picking a specific version here is that it gets pulled from upstream quickly and then your build breaks. Mostly I noticed this with the gh cli

Reply:
We can always have renovate wait a few days to help I guess?